### DISCUSSION TITLE
Disabling Automatic User Creation for External Identity Provider

### DISCUSSION THREAD

**Discussion by taseaford** [abpframework/abp#7873] [Q&A]
We have setup an ABP project as an Identity federator between some internal applications and an enterprise authentication provider. One legacy client applications that we are currently trying to integrate has it's own user DB. We want to use the identity server to verify authentication and then pass the result on to the application without storing that user in the identity DB since the application already has it's own.

Can anyone point me in the direction of what to override in order to prevent the automatic creation of a user within the identity DB?

**Comment by maliming** [comment]
You can check this : 
https://github.com/abpframework/abp/blob/bcefd4590520e862cfbd20ce865991ce3316e720/modules/account/src/Volo.Abp.Account.Web/Pages/Account/Register.cshtml.cs#L81-L91
